flake8_tidy_imports.py

```diff
@@ -132,3 +135,247 @@ def rule_I201(self, node):
+        'mhlib': 'moved in Python 3. Use mailbox instead',
+        'mimetools': 'moved in Python 3. Use email instead',
+        'MimeWriter': 'moved in Python 3. Use email instead',
+        'mimify': 'moved in Python 3. Use email instead',
+        'multifile': 'moved in Python 3. Use email instead',
+        'mutex': 'removed in Python 3',
+        'new': 'removed in Python 3',
+        'os.getcwd': 'moved in Python 3. six.moves.getcwdb can be used as a drop-in replacement',
+        'os.getcwdu': 'moved in Python 3. six.moves.getcwd can be used as a drop-in replacement',
+        'pipes.quote': 'moved in Python 3. six.moves.shlex_quote can be used as a drop-in replacement',
+        'platform._bcd2str': 'removed in Python 3',
+        'platform._mac_ver_gstalt': 'removed in Python 3',
+        'platform._mac_ver_lookup': 'removed in Python 3',
+        'plistlib.readPlist': 'moved in Python 3. Use plistlib.load instead',
+        'plistlib.readPlistFromBytes': 'moved in Python 3. Use plistlib.loads instead',
+        'plistlib.writePlist': 'moved in Python 3. Use plistlib.dump instead',
+        'plistlib.writePlistToBytes': 'moved in Python 3. Use plistlib.dumps instead',
+        'popen2': 'moved in Python 3. Use subprocess instead',
+        'posixfile': 'moved in Python 3. Use fcntl.lockf instead',
+        'pure': 'removed in Python 3',
+        'pydoc.Scanner': 'removed in Python 3',
+        'Queue': 'moved in Python 3. six.moves.queue can be used as a drop-in replacement',
+        'repr': 'moved in Python 3. six.moves.reprlib can be used as a drop-in replacement',
+        'rexec': 'removed in Python 3',
+        'rfc822': 'moved in Python 3. Use email instead',
+        'robotparser': 'moved in Python 3. six.moves.urllib.robotparser can be used as a drop-in replacement',
+        'ScrolledText': 'moved in Python 3. six.moves.tkinter_scrolledtext can be used as a drop-in replacement',
+        'sgmllib': 'removed in Python 3',
+        'sha': 'moved in Python 3. Use hashlib instead',
+        'sha.new': 'moved in Python 3. hashlib.sha1 can be used as a drop-in replacement',
+        'sha.sha': 'moved in Python 3. hashlib.sha1 can be used as a drop-in replacement',
+        'SimpleDialog': 'moved in Python 3. six.moves.tkinter_simpledialog can be used as a drop-in replacement',
+        'SimpleHTTPServer': 'moved in Python 3. six.moves.SimpleHTTPServer can be used as a drop-in replacement',
+        'SimpleXMLRPCServer': 'moved in Python 3. six.moves.xmlrpc_server can be used as a drop-in replacement',
+        'smtplib.SSLFakeFile': 'moved in Python 3. Use socket.socket.makefile instead',
+        'SocketServer': 'moved in Python 3. six.moves.socketserver can be used as a drop-in replacement',
+        'sre': 'moved in Python 3. Use re instead',
+        'statvfs': 'moved in Python 3. Use os.statvfs instead',
+        'string.atof': 'moved in Python 3. float can be used as a drop-in replacement',
```

Comment:
so one problem with all of this is that we are only detecting import statements, like `from string import atof`. Using `import string` then later `string.atof()` is not detected. Not sure if we should be fixing this in a library called `flake8-tidy-imports`...